plugins/elantp/elantp.quirk

```diff
@@ -26,6 +26,16 @@ Flags = elantp-recovery
 [513cde3d-d939-59bd-a634-5c1645ebb93b]
 Flags = elantp-recovery
 
+[I2C\MODALIAS_acpi:ELAN0000:]
```

Comment:
need to double check the ending `:`

Reply:
> need to double check the ending `:`

Yes, the `:` at the end is necessary after the test result

Reply:
thanks!